packages/command-palette/src/commands.tsx

```diff
@@ -1744,16 +1744,11 @@ export function useCommands() {
 					),
 				].join( KEYWORD_SEPARATOR ),
 				context: [ '/settings', '/wp-admin/options-' ],
-				callback: ( params ) =>
-					commandNavigation(
-						params.site?.is_wpcom_atomic && siteUsesWpAdminInterface( params.site )
-							? '/wp-admin/options-general.php?page=page-optimize'
-							: '/settings/performance/:site'
-					)( params ),
+				callback: commandNavigation( '/sites/settings/performance/:site' ),
 				siteSelector: true,
 				siteSelectorLabel: __( 'Select site to manage performance settings', __i18n_text_domain__ ),
-				capability: SiteCapabilities.MANAGE_OPTIONS,
 				filterP2: true,
+				...siteFilters.hostingEnabled,
```

Comment:
Require hosting features for performance options - treatment group simple site was redirecting to old and nearly empty /settings/performance page.